README.md

```diff
@@ -4,9 +4,9 @@
 [![Build Status](https://github.com/ViaVersion/ViaBackwards/actions/workflows/gradle.yml/badge.svg?branch=master)](https://github.com/ViaVersion/ViaBackwards/actions)
 [![Discord](https://img.shields.io/badge/chat-on%20discord-blue.svg)](https://viaversion.com/discord)
 
-**Allows the connection of older clients to newer server versions for Minecraft servers.**
```

Comment:
The old description is not wrong and sounds better imo; and otherwise also needs updating in the build script (build.gradle.kts).